ct/outline.sh

```diff
@@ -57,8 +57,9 @@ header_info
 check_container_storage
 check_container_resources
 if [[ ! -d /opt/outline ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
-RELEASE=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
-if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+RELEASE_TAG=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+RELEASE=${RELEASE_TAG#v}
```

Comment:
why? you can just use one of this:
RELEASE=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
=> 0.81.1

Reply:
Maintainability and understandability: 
```
RELEASE=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
``` 
will not tell you that it gets a tag name and you strip the v-prefix from it.
```
RELEASE_TAG=$(curl -s https://api.github.com/repos/outline/outline/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
RELEASE=${RELEASE_TAG#v}
```
does, but in context of  https://github.com/community-scripts/ProxmoxVE/pull/330 this will even allow something like
```
RELEASE_TAG=$(get_latest_github_release_tag outline outline)
RELEASE=${RELEASE_TAG#v}
```
where it is even clearer. To say it with the Clean Code author: 
> “Indeed, the ratio of time spent reading versus writing is well over 10 to 1. We are constantly reading old code as part of the effort to write new code. ...[Therefore,] making it easy to read makes it easier to write.”

Reply:
Why is it important to know that a `v` prefix is stripped? A github release tag can be anything, we are just interested in the semantic version which always should be stored in `RELEASE`. Follow conventions and lets keep discussion on changes to how github releases should be handled in #330

Reply:
Since you ask: https://en.wikipedia.org/wiki/Zen_of_Python line 2: "explicit is better than implicit"
But I'll make the changes as requested, and continue the discussion elsewhere.